frontend/src/static/js/main.js

```diff
@@ -250,7 +250,11 @@ window.app = new window.Vue({
       return JSON.stringify(dataObj);
     },
 
-    getRepoSenseLink() {
+    getRepoSenseMainSiteLink() {
```

Comment:
`getRepoSenseMainSiteLink` -> `getRepoSenseHomeLink` will be more appropriate.
MainSite is like suggesting you have other mirror sites

Reply:
Didn't think of it this way. Thank you.